src/main/java/ch/njol/skript/expressions/ExprExplodedBlocks.java

```diff
@@ -68,6 +75,25 @@ protected Block[] get(Event e) {
 		List<Block> blockList = ((EntityExplodeEvent) e).blockList();
 		return blockList.toArray(new Block[blockList.size()]);
 	}
+
+	@Override
+	public @Nullable Class<?>[] acceptChange(ChangeMode mode) {
+		if (mode == ChangeMode.REMOVE)
```

Comment:
Are there other operations we could support? At least `DELETE` would work, as in `clear the exploded blocks`?

Reply:
actually would be good to support the full set of `SET`, `ADD`, `REMOVE`, `DELETE`

Reply:
oh yeah that would be a good idea. Ill whip that up and test it